teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -2,6 +2,38 @@
 
 Reviewers are in charge of **curriculum teams.** Each team consists of 4-6 people who work on different modules of curriculum. They will be tasked with a project, usually covering a topic, and a deadline. By this deadline, all activities, labs and workshops for that project should be completed.
 
+## Developer
+
```

Comment:
generally speaking there should be a chronological basis for each section. Mention Fri/Sun deadlines, Tues deadline for my review, etc. etc.